tests/console/znc.pm

```diff
@@ -17,7 +17,7 @@ sub run {
     select_console('root-console');
     zypper_call('in znc');
 
-    script_run("sudo -u znc znc --makeconf | tee /dev/$serialdev; echo zncconf-status-\$? > /dev/$serialdev", 0);
```

Comment:
This will break the test. This "hack" is used to run application in background and wait till it exits.
If you do it like this, you need to adjust code on L#79 `wait_serial("zncconf-status-0") || die "'znc --makeconf' could not finish successfully";` where we wait for this message once znc returns.

Reply:
Indeed - I did it differently now. Best option is probably to add a new "assert_serial" function...